### PR TITLE
Replace hard code currency symbol

### LINKED ISSUE #180: "Add to cart" currency symbol
Hi,

The" add to cart" button is rendered with an hard-codded currency symbol : 

```${this.getPrice()}```

it should be : 

```{`this.getCurrencySymbol(product.price.formatted_with_symbol)}{this.getPrice()}```

maybe add it in utils : 

`getCurrencySymbol(priceFormattedWithSymbol) {
    return priceFormattedWithSymbol.substring(1, 0);
  }`

### PR DESCRIPTION
**Pull request by zb2oby**
replace the hard coded currency symbol on "Add to cart" button

Fixes #180

### CODE REVIEW

**Comment by zb2oby** [comment]
linked to #180

File: components/productAssets/ProductDetail.js [hunk 1]
```diff
@@ -139,7 +147,7 @@ class ProductDetail extends Component {
               Add to cart
             </span>
             <span className="border-left border-color-white pl-3">
-            ${this.getPrice()}
+            {priceSymbol}{this.getPrice()}
```
Comment (robbieaverill) [inline components/productAssets/ProductDetail.js:150]: I'd recommend using `price.formatted_with_symbol` here. I don't think the `getPrice()` function is actually needed at all, we should be able to get the price from the API. @jaepass do you know why `getPrice()` exists here?

Comment (jaepass) [reply components/productAssets/ProductDetail.js:150]: Hey I'm just looking at it and looks like the `getPrice()` function is to display the total variant price with base price added with variant option selected?

Comment (zb2oby) [reply components/productAssets/ProductDetail.js:150]: I think this getPrice is here to calculate the price computed with the variant amount selected for the product. The "product.price" is just the price without variant amount.

**Review by jaepass — APPROVED** [review]

**Pull request merged**